src/python/genny/genny_auto_tasks.py

```diff
@@ -52,12 +52,27 @@ def get_project_root():
     try:
         out = subprocess.check_output('git rev-parse --show-toplevel', shell=True)
     except subprocess.CalledProcessError as e:
-        print(e.output)
+        print(e.output, file=sys.stderr)
         raise e
 
     return out.decode().strip()
 
 
+def validate_user_workloads(workloads):
+    if len(workloads) == 0:
+        return 'No workloads specified'
+
+    genny_root = get_project_root()
+    for w in workloads:
+        workload_path = '{}/src/workloads/{}'.format(genny_root, w)
+        if not os.path.isfile(workload_path):
+            return 'no file at path {}'.format(workload_path)
+        if not workload_path.endswith('.yml'):
+            return '{} is not a .yml workload file'.format(workload_path)
+
+    return None
```

Comment:
if the user passes in multiple invalid workloads it would be nice to know of all of them rather than just the first one. I think return an array of error messages (empty array if no errors) and print all of them and exit if non-empty from the caller.

Reply:
Done.